### DISCUSSION TITLE
week 5 was hard... what should I do ?

### DISCUSSION THREAD

**Discussion by MaelaKloareg** [ColinFay/ur-first-5k#310] [Q&A]
Hi @ColinFay and the others.

So everything went very well til the end of week 4, and last week was harder. For example, today (end of my week 5), I ran 12 mins, walked 3, and then ran 13, instead of running 30 mins. I'm confused because a week ago I ran 22 minutes easily for W04D7. 

Is this normal ? Do you have any suggestions ? Should I just feel free to walk when I feel I need it, and try to continue with the proposed exercices, or should I run less, or ... 

Other question : how would you define "steady pace" compared to "easy run"/"recovery pace". My watch gives my my heart rate, so I cn see the difference after running, but I was a bit to ambitious on my "steady pace" of my W05D2.

Anyway, I try to keep having fun, seems the most important to me to keep motivated ;-).

**Comment by ColinFay** [comment]
Hey @MaelaKloareg 

Yeah it can definitely happen. Sometimes, it can be linked to a more general tiredness of the body (that can be caused by some lack of sleep, food, maybe it can even be linked to the weather — for example it's harder to run when it's very cold / very hot). For women, it can also be linked to the body cycle (good read on this: https://trailrunnermag.com/training/menstrual-cycle-impacts-running-can.html). 

And sometimes it's just that you don't feel like it. I've been running for more than a decade now and from time to time I have bad weeks when I don't feel like I have enough energy :) 
The most important is to not let this get into the way of your training. 
It's usually a temporary feeling, and it gets better. 

If you feel low during a running, you should definitely feel free to walk. It's better to complete the distance than to exhaust yourself trying to push :) 

For your other question, "easy pace" is the pace at which you'd still be able to speak, have a conversation. Recovery pace is a little slower than that, and steady pace is a "fast" pace, i.e one where you wouldn't be able to have a conversation, but also one you should be able to handle for a couple of minutes :)

Hope this helps! 

Colin

**Reply by MaelaKloareg** [reply]
Thank you @ColinFay for your answers, it helps a lot!
I've managed my W06D2 without problem : it wasn't as easy as weeks 1 to 4 but definitely easier thant week 5, so I have hope ;-) !
